### ISSUE TITLE
add "max loggings per username" ability

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1633]
### Description

currently, we have max logging per ip adress and logging delay per ip adress feechers to limit with: i have a request that please add "max loggings per username" feecher also, because, if we are able  to limit people with ip adress, people can really eezili change their ip adress and continu to do their job, but they can't change their username because only admins can add or remove accounts, so limitting people with username is much bettar and it is much safe than limitting with ip adress: please add max logging per username feecher, but you don't need to add logging delay per username, because we already able to do that one in user accounts list abuse privention tab: tt vertion: 5.11 thanks

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by bear101** [comment]
If you in "User Accounts" uncheck "User can log in multiple times" then only one username can be active on the server.

**Comment by amirmahdifard** [comment]
@bear101 you are right, but here is something remaning: i want to give the moltiple time acces to the users, but i want to limit them so that they only can logging up to 5 clients, so in this kase, we need max logging per ip adress.
